cmsimple/classes/Page.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace XH;
+
+/**
+ * Initiates a page object. Its methods act as a wrapper
+ * of the XH/Pages methods for better OOP access
```

Comment:
This should be `XH\Pages` instead of `XH/Pages`. Perhaps `use XH\Pages` would also be sensible, in which case `Pages` might suffice here (not sure how that's handled by Doxygen).